app/main/index.tsx

```
@@ -313,29 +52,42 @@ export default function LobbyScreen() {
     setIsTransitioning(true);
   };
 
-  async function startAI() {
-    await dispatch(createVsAI('300+0')).unwrap();
+  function startAI() {
+    console.log('Début du jeu contre l’IA');
     dispatch(setMode('ai'));
-    router.push('/main/game');
+    socket.emit('createVsAI', { timeControl: '300+0' });
+    socket.once('aiGameCreated', (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)